pygmt/tests/test_clib_virtualfiles.py

```diff
@@ -321,16 +326,21 @@ def test_virtualfile_from_matrix_slice(dtypes):
 
 def test_virtualfile_from_vectors_pandas(dtypes):
     """
-    Pass vectors to a dataset using pandas Series.
+    Pass vectors to a dataset using pandas Series, checking both numpy and
+    pyarrow dtypes.
     """
     size = 13
+    if pa is not None:
+        dtypes.extend([f"{dtype}[pyarrow]" for dtype in dtypes])
```

Comment:
Can we modify the `fixture_dtypes` fixture directly, so that pyarrow types are more extensively tested?

Reply:
So `fixture_dtypes` is also used by some unit tests that use `np.array`, but we can't create numpy arrays with pyarrow dtypes.